### PR TITLE
Sqlite Cleanup and config vars update

### PR DESCRIPTION
**Pull request by ahetawal-p**
Hey @hyandell 
Here is the PR for cleanup of remaining sqlite stuff.

- Tried cleaning the queries under `syn-tng` as they were not postgres compliant. Note: Haven't tested these changes, you might to run them once on your side.

- Fixed `util.rb` to remove hardcoding of DB config

- Updated config var reading for db, logins and organizations from env variables if someone doesn't want to checkin this info in config file on repo.

Let me know what you think..

### CODE REVIEW

**Comment by hyandell** [comment]
Cool - and you found that user->username bug too. I just got bit by that trying to run against a remote RDS instance. The sync-tng code was mostly working, but I do have a bug on login loading happening so it'll be interesting to see if this fixes that.

To cover the licensing side of things - can you confirm this contribution is under the terms of the Apache 2.0 license?

**Comment by ahetawal-p** [comment]
Yes, it is under terms of Apache 2.0 license.

**Comment by ahetawal-p** [comment]
Also, please let me know if you can merge this, so I can update my other Heroku branch to keep it in sync and cleaned up !

File: github-sync-tng/issue_command.rb [hunk 1]
```diff
@@ -93,16 +95,18 @@ def get_labels(client, db, orgrepo)
...
+        # Fill Labels again
+        labels.each do |label|
+          db["INSERT INTO labels (orgrepo, url, name, color) VALUES (?, ?, ?, ?)", orgrepo, label.url, label.name, label.color].insert
+        end
+      end
+    rescue => e
+      puts "Error during processing: #{$!}"
     end
-    db.execute("END TRANSACTION")
-  end
```
Comment (hen) [inline github-sync-tng/issue_command.rb]: Mental note that this 'end' is still needed. Will fix after merging.

File: github-sync-tng/issue_command.rb [hunk 2]
```diff
@@ -129,26 +133,25 @@ def sync_items(queue, context, issue_db, org, repo)
...
-        db_fix_merged_at(issue_db, context.client, issue_block, org, repo)           # Put in PR specific data - namely merged_at
-        db_add_pull_request_files(issue_db, context.client, issue_block, org, repo)  # Put in PR specific data - namely the files + their metrics
-        issue_db.execute("END TRANSACTION");
-
-        # remove the block just loaded
-        issues=issues.drop(BLOCK_SIZE)
       end
 
     end
-  end
```
Comment (hen) [inline github-sync-tng/issue_command.rb]: Mental note that this 'end' is still needed. Will fix after merging.

**Comment by hen** [comment]
Aiming to merge Monday.

Tested tonight and it does fix the bug I was going to have to fix in the metadata code :) Logins pull happily now in the xsync mode.

**Comment by ahetawal-p** [comment]
Fixed the missing ends, thanks for catching them. Glad it all worked for you.

**Comment by hyandell** [comment]
Merged :)

**Pull request merged**